end_to_end_tests/client_test.py

```diff
@@ -190,5 +193,92 @@ def test_get_sigmarule(self):
         count = len(data_frame)
         self.assertions.assertEqual(count, 1)
 
+    def test_add_event_attributes(self):
+        """Tests adding attributes to an event."""
+        sketch = self.api.create_sketch(name="Add event attributes test")
+        sketch.add_event("event message", "2020-01-01T00:00:00", "timestamp_desc")
+        # Wait for new timeline and event to be created
+        time.sleep(1)
```

Comment:
This looks like it can create flaky tests. Could we check that the timeline is ready by looking at the status of it? IF not easily done, we could afford to increase the time at least, say ~5s to make sure the datastore is ready.

Reply:
Added a retry loop to check the status.